build/build.go

```diff
@@ -575,7 +575,7 @@ func NewMappingCallback(m *sourcemap.Map, goroot, gopath string) func(generatedL
 	}
 }
 
-func jsFilesFromDir(dir string) ([]string, error) {
+func JsFilesFromDir(dir string) ([]string, error) {
```

Comment:
(This sort of thing would fit well in a Developers Guidelines, IMHO... Perhaps I should propose the development and formalization of this in an issue)